Document/0x05b-Basic-Security_Testing.md

```diff
@@ -107,7 +107,9 @@ For a typical mobile app security build, you'll usually want to test a debug bui
 
 #### Recommended Tools
 
--- ToDo: <https://github.com/OWASP/owasp-mstg/issues/1227>
+This section introduces some of the most popular and useful tools and frameworks for security testing Android applications that you will be using throughout this guide. For each tool, we give a brief introduction, a reference to the installation instructions, some handy use cases and examples, and links to the corresponding command documentation.
+
+See the section [Tools for Android Security Testing](#Tools "Android Security Testing tools") for the full list of covered tools.
```

Comment:
Please have a look at https://github.com/OWASP/owasp-mstg/blob/master/Document/0x06b-Basic-Security-Testing.md#recommended-tools---ios-device: what we are looking for, is a bit more of an introduction into the tools and a listing, like we did over there. That way, the reader has the same experience in the Android chapter as in the iOS chapter ;-).

Reply:
I agree and have two things to point out:
- I've open a PR #1328 that removes the #Tools "Android Security Testing tools" section because it requires an extra effort to maintain and we have all recommended tools in this section anyway. Please check the PR and pull from master to have the latest changes.
- I also think we should give the reader the same experience in the Android chapter as in the iOS chapter. Regarding this, it might be a good idea to mimic what we have on that chapter: #### Recommended Tools - Android Device and #### Recommended Tools - Host Computer

Reply:
I also think we should give the reader the same experience in the Android chapter as in the iOS chapter. Regarding this, it might be a good idea to mimic what we have on that chapter: 

#### Recommended Tools - Android Device
#### Recommended Tools - Host Computer